tools/executors/serve/impl.js

```diff
@@ -0,0 +1,366 @@
+'use strict';
```

Comment:
this is a generated file by tsc, from the `./impl.ts` file

Reply:
removed it from the repo and added a step to the postinstall step to have it compiled.